src/app/+item-page/simple/field-components/specific-field/generic-uri/item-page-generic-uri-field.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, Input } from '@angular/core';
+import {ItemPageFieldComponent} from '../item-page-field.component';
+import {Item} from '../../../../../core/shared/item.model';
+
+@Component({
+  selector: 'ds-item-page-generic-uri-field',
+  templateUrl: './item-page-generic-uri-field.component.html'
+})
+/**
+ * This component can be used to represent any uri on a simple item page.
+ * It is the most generic way of displaying uris
+ * It expects 4 parameters: The item, a separator, the metadata keys and an i18n key
+ */
+export class ItemPageGenericUriFieldComponent extends ItemPageFieldComponent {
```

Comment:
@benbosman : Looking at the code here, I was surprised to see this isn't actually used anywhere.  Is there a reason why we don't *replace* the current `ItemPageUriFieldComponent` with the new generic `ItemPageGenericUriFieldComponent`?   Do we need two different URI components?